bitcoinspv/types/btccache.go

```diff
@@ -8,9 +8,9 @@ import (
 
 // BTCCache represents a thread-safe cache of indexed blocks
 type BTCCache struct {
-	sync.RWMutex
-	maxEntries int64
 	blocks     []*IndexedBlock
+	maxEntries int64
+	sync.RWMutex
```

Comment:
The private / inherit type should be first.

Reply:
Done